### PR TITLE
plex-desktop: Add version 1.30.1.2115-81e1fc3f

### PR DESCRIPTION
**Pull request by jaysonsantos**

### CODE REVIEW

File: bucket/plex-desktop.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.29.1.1974-a65b3ef3",
+    "description": "Plex for Windows.",
```
Comment (Ash258) [inline bucket/plex-desktop.json:3]: - How is this different from plex player?
- Dot at the end does not make sense

File: bucket/plex-desktop.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "homepage": "https://www.plex.tv/apps-devices/#modal-devices-plex-for-windows",
+    "license": "Proprietary",
```
Comment (Ash258) [inline bucket/plex-desktop.json:5]: https://www.plex.tv/media-server-downloads/?platform=windows#remodal-terms

File: bucket/plex-desktop.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        "url": "https://plex.tv/pms/downloads/6.json",
+        "jsonpath": "$.computer.Windows.version"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://downloads.plex.tv/plex-desktop/$version/windows/Plex-$version-x86_64.exe#/dl.7z",
+                "hash": {
+                    "url": "https://plex.tv/pms/downloads/6.json",
+                    "jsonpath": "$.computer.Windows.releases.checksum"
```
Comment (Ash258) [inline bucket/plex-desktop.json:28]: Add condition for build `windows-x86_64`

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/plex-desktop.json (hunk 2 above)
Comment (jaysonsantos) [reply bucket/plex-desktop.json:5]: you mean to change Proprietary to the URL?

File: bucket/plex-desktop.json (hunk 1 above)
Comment (jaysonsantos) [reply bucket/plex-desktop.json:3]: https://www.plex.tv/blog/desktop-af/
"...It has all the glorious player capability of Plex Media Player, plus an exciting new offline capability..."

File: bucket/plex-desktop.json (hunk 3 above)
Comment (jaysonsantos) [reply bucket/plex-desktop.json]: i've rebase it witht he suggestion + a new version that was released

File: bucket/plex-desktop.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        "url": "https://plex.tv/pms/downloads/6.json",
+        "jsonpath": "$.computer.Windows.version"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://downloads.plex.tv/plex-desktop/$version/windows/Plex-$version-x86_64.exe#/dl.7z",
+                "hash": {
+                    "url": "https://plex.tv/pms/downloads/6.json",
+                    "jsonpath": "$.computer.Windows.releases[?(@.build == \"windows-x86_64\")].checksum"
```
Comment (Ash258) [inline bucket/plex-desktop.json:28]:
```suggestion
                    "jsonpath": "$.computer.Windows.releases[?(@.build == 'windows-x86_64')].checksum"
```

File: bucket/plex-desktop.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.30.1.2115-81e1fc3f",
+    "description": "Plex for Windows",
+    "homepage": "https://www.plex.tv/apps-devices/#modal-devices-plex-for-windows",
+    "license": "Proprietary",
+    "architecture": {
+        "64bit": {
+            "url": "https://downloads.plex.tv/plex-desktop/1.30.1.2115-81e1fc3f/windows/Plex-1.30.1.2115-81e1fc3f-x86_64.exe#/dl.7z",
+            "hash": "c9eb3ecb157d32251abaaae3a7b6b8aca1d51f3846ee29f1dcfe2b10ab96b11c"
```
Comment (Ash258) [inline bucket/plex-desktop.json:9]:
```suggestion
            "url": "https://downloads.plex.tv/plex-desktop/1.30.1.2115-81e1fc3f/windows/Plex-1.30.1.2115-81e1fc3f-x86_64.exe#/dl.7z",
            "hash": "sha1:9fd30e82fe8dad74f58edf1f6ba9f1688f4e408a"
```

File: bucket/plex-desktop.json [hunk 6]
```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.30.1.2115-81e1fc3f",
+    "description": "Plex for Windows",
+    "homepage": "https://www.plex.tv/apps-devices/#modal-devices-plex-for-windows",
+    "license": "Proprietary",
```
Comment (Ash258) [inline bucket/plex-desktop.json:5]:
```suggestion
    "license": {
        "identifier": "Proprietary",
        "url": "https://www.plex.tv/media-server-downloads/?platform=windows#remodal-terms"
    },
    "suggest": {
        "vcredist": "vcredist2019"
    },
```

File: bucket/plex-desktop.json [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://downloads.plex.tv/plex-desktop/1.30.1.2115-81e1fc3f/windows/Plex-1.30.1.2115-81e1fc3f-x86_64.exe#/dl.7z",
+            "hash": "c9eb3ecb157d32251abaaae3a7b6b8aca1d51f3846ee29f1dcfe2b10ab96b11c"
+        }
+    },
+    "shortcuts": [
```
Comment (Ash258) [inline bucket/plex-desktop.json:12]:
```suggestion
    "pre_install": "Remove-Item \"$dir\\`$*\", \"$dir\\Uninst*\", \"$dir\\vc_red*\" -Recurse",
    "shortcuts": [
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/plex-desktop.json (hunk 5 above)
Comment (jaysonsantos) [reply bucket/plex-desktop.json:9]: Could you say what is the criteria to use sha1 instead of sha2?

Comment (Ash258) [reply bucket/plex-desktop.json:9]: The provided hashes are sha1..

Comment (jaysonsantos) [reply bucket/plex-desktop.json:9]: Ah, I got it.
`checkver`'s response is a sha1

**Review by Ash258 — CHANGES_REQUESTED** [review]
https://github.com/lukesampson/scoop-extras/pull/5935#discussion_r606949072

**Comment by jaysonsantos** [comment]
> 
> 
> [#5935 (comment)](https://github.com/lukesampson/scoop-extras/pull/5935#discussion_r606949072)

This is here https://github.com/lukesampson/scoop-extras/pull/5935/files#diff-5c19d32eb732170054d8d3a3d7a349494c46d404dce27a335a01027d0ffac8ffR18

Also, when I commited I put you as a co-author, doesn't that work as well?

**Review by Ash258 — APPROVED** [review]

**Pull request merged**